http_cache/esi.rst

```diff
@@ -172,7 +172,7 @@ of the main page::
     // ...
     class NewsController extends AbstractController
     {
-        public function latest($maxPerPage)
+        public function latest($maxPerPage): Response
```

Comment:
While here can you perhaps typehint the $maxPerPage as well?

Reply:
Done !